src/main/java/seedu/address/model/person/ModuleRoleContainsKeywordsPredicate.java

```diff
@@ -7,7 +7,7 @@
 import seedu.address.commons.util.ToStringBuilder;
 
 /**
- * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
+ * Tests that a {@code Person}'s {@code ModuleRoleMap} matches any of the keywords given.
```

Comment:
Thanks for noticing this!

Reply:
No problem! I think I spot similar mistakes in the past so everyone reading this please take note of this potential mistake in comment when copying pasting! Thanks!